contrib/charts/bivac/templates/configmap.yaml

```diff
@@ -10,6 +10,9 @@ metadata:
     heritage: {{ .Release.Service }}
 data:
   providers-config.toml: |
+    {{- if .Values.providers -}}
```

Comment:
suggestion: instead of doing a if/else to overwrite the default configuration, I'd rather set the default configuration in the value `providers`. The end result is the same, but the implementation is simpler.